src/odemis/gui/comp/viewport.py

```diff
@@ -1193,7 +1193,7 @@ def __init__(self, *args, **kwargs):
 
         self.canvas.Bind(wx.EVT_MIDDLE_DOWN, self.on_drag_start)
         self.canvas.Bind(wx.EVT_MIDDLE_UP, self.on_drag_end)
-        # self.canvas.Bind(wx.EVT_MOTION, self.on_canvas_motion)
+        self.canvas.Bind(wx.EVT_MOTION, self.on_canvas_motion)
```

Comment:
Would you mind not commenting out this line in the previous commit, so that this commit doesn't have to uncomment it? It'll be easier to backport....

Reply:
Hmm..... how do I do this.... I think the commented-out line was a mistake. Maybe it is easier to back-port all of these changes at once?

Reply:
I can do it with you, and show you. Essentially, you'd discard the commit history, and recreate each commit, using "git add -i".